app/assets/javascripts/select-kit/addon/components/category-chooser.js

```diff
@@ -94,8 +92,10 @@ export default ComboBoxComponent.extend({
   search(filter) {
     if (this.site.lazy_load_categories) {
       return Category.asyncSearch(this._normalize(filter), {
-        scopedCategoryId: this.selectKit.options?.scopedCategoryId,
-        prioritizedCategoryId: this.selectKit.options?.prioritizedCategoryId,
+        includeUncategorized: this.selectKit.options.allowUncategorized,
+        rejectCategoryIds: [this.selectKit.options.excludeCategoryId],
+        scopedCategoryId: this.selectKit.options.scopedCategoryId,
```

Comment:
This is used to filter only subcategories of `scopedCategoryId` and I have not found a use case yet to filter categories for multiple parent category IDs.

Reply:
Ah my bad, I got confused here. Since `rejectCategoryIds` in the backend controller is being checked now that it's an array, I assumed `scopedCategoryId` was the same, but it's not, it's `select_category_ids`. All good, I think...

Reply:
I'd prefer `parentCategoryId` instead of `scopedCategoryId` too, but that might break themes that use it. Maybe we will deprecate this when we will be removing the old sync category code?